generators/kubernetes/templates/registry/_jhipster-registry.yml

```diff
@@ -84,17 +84,13 @@ spec:
         # Registry configuration
         - name: SPRING_PROFILES_ACTIVE
           value: prod,native
-        - name: SECURITY_USER_PASSWORD
+        - name: JHIPSTER_REGISTRY_PASSWORD
           valueFrom:
             secretKeyRef:
               name: registry-secret
               key: registry-admin-password
         - name: JHIPSTER_SECURITY_AUTHENTICATION_JWT_SECRET
           value: my-secret-token-to-change-in-production
-        - name: EUREKA_CLIENT_FETCH_REGISTRY
-          value: 'true'
-        - name: EUREKA_CLIENT_REGISTER_WITH_EUREKA
-          value: 'true'
```

Comment:
Isn't it the default in Peer-aware mode? In Peer-aware mode, a Eureka server also acts as a Eureka client and registers to a peer on a given URL. EUREKA_CLIENT_REGISTER_WITH_EUREKA needs to be true in this case. In standalone mode, this can be set to false.